install_builder/deadline-cloud-for-keyshot.xml

```diff
@@ -117,7 +95,6 @@
             <value>Deadline Cloud for KeyShot
 - Install the integrated KeyShot submitter files to the installation directory
```

Comment:
`Install the integrated KeyShot submitter files to the installation directory` should be removed as well since the only file being "installed" is the `Submit to AWS Deadline Cloud` script mentioned on the line below.